src/model_signing/verifying.py

```diff
@@ -97,7 +97,9 @@ def verify(
 
         if self._hashing_config is None:
             self._guess_hashing_config(expected_manifest)
-        actual_manifest = self._hashing_config.hash(model_path)
+        actual_manifest = self._hashing_config.hash(
+            model_path, expected_manifest.ignore_paths
+        )
```

Comment:
handling manifest ignore paths could likely fall under the `if self._hashing_config is None` block instead.

Reply:
This part has changed now.